ui/src/keyboardLayouts/en_UK.ts

```diff
@@ -0,0 +1,107 @@
+import { KeyCombo } from "../keyboardLayouts"
```

Comment:
Could this (or the US) be synthesized from the other one (just overlaying as is done for _Français de Suisse_ deriving from _Français_? Could also do so for German and Swiss German.

Reply:
Sure, we could build an inheritance tree to see which keyboard layouts are variants of the others. But it seems a lot of effort to maintain and would require a lot of brain muscle to remember the dependencies when future changes are being made to not introduce regressions, I think.

But I‘m open to it, feel free to send a PR.